src/components/Livestream.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components'
+import { Card } from '../components/Common.js'
+import { H2 } from '../components/Typography'
+
+const TwitchFrame = styled.iframe`
+  border-radius: 5px;
+  border: none;
+`
+
+const LivestreamContainer = styled(Card)`
+  text-align: center;
+`
+
+const LeftAlign = styled.div`
+  width: 1280px;
+  text-align: left;
+  margin: 0 auto;
+`
+
+const Pulse = keyframes`
+  0% {
+    background-color: #ff0000;
+  }
+  50% {
+    background-color: #ff000000;
+  }
+  100 {
+    background-color: #ff0000;
+  }
+`
+
+const LiveDot = styled.span`
+  height: 15px;
+  width: 15px;
+  background-color: #f00;
+  border-radius: 50%;
+  display: inline-block;
+  animation: ${Pulse} 1.5s ease infinite;
+`
+
+const Livestream = () => {
+  return (
+    <LivestreamContainer>
+      <TwitchFrame
+        src="https://player.twitch.tv/?channel=nwplus&parent=localhost&parent=live.nwplus.io"
+        height="720"
+        width="1280"
+        frameborder="0"
+        scrolling="no"
+        allowfullscreen="true">
+      </TwitchFrame>
+      <LeftAlign>
+        <H2><LiveDot /> nwPlus - Live</H2>
+                View on <a href="https://www.twitch.tv/nwplus" target="_blank" rel="noopener noreferrer">Twitch</a>
```

Comment:
Whats the usecase for viewing on twitch?

Reply:
![image](https://user-images.githubusercontent.com/23178940/92332600-fdc89200-f033-11ea-8b19-ebd6b094e558.png)

I just wanted something to put here LOL should I just omit it then

Reply:
uhmmm i guess omit for now